File: veins_test/deno/stdin_upper.ts
#!/usr/bin/env -S fragletc --vein=deno
const buf = new Uint8Array(1024);
while (true) {
  const n = await Deno.stdin.read(buf);
  if (n === null) break;
  const s = new TextDecoder().decode(buf.subarray(0, n));
  console.log(s.trim().toUpperCase());
}

File: veins_test/java/stdin_upper.java
#!/usr/bin/env -S fragletc --vein=java
import java.util.*;

public class Fraglet {
    public static void main(String[] args) {
        java.util.Scanner s = new java.util.Scanner(System.in);
        while (s.hasNextLine()) {
            System.out.println(s.nextLine().toUpperCase());
        }
    }
}
